### ISSUE TITLE
Can't import npm module both on server and on client

### ISSUE THREAD

**Issue by eric-burel** [meteor/meteor#7040]
Hello,

I am using Meteor 1.3 and the jStat package, installed from the command `meteor install --save jStat`.

I am trying to import jstat in a file (directory : `imports/math/rates.js`). 
The syntax `import { jStat } from 'jStat' ` does work on server side, but not on client side : jStat is undefined in the browser.

The syntax `import jStat from jStat' gives an empy object `{}` on the client side, but jStat.jStat is correctly defined.

So, I tried the `import 'jStat'` syntax.. which work on the client side, but not on the server side : jStat is undefined on the server.

To sum it up, in the `/imports/math/rates.js` :
`import { jStat } from 'jStat';` defines jStat on server only 
`import jStat from 'jStat';` defines jStat.jStat on server but set jStat to {} on client
`import 'jStat'` defines jStat on client only.

It seems that jStat exports a global object jStat.jStat. I did install `meteor npm install --save meteor-node-stubs` when upgrading to 1.3.

Am I missing something ?

**Comment by abernix** [comment]
Make sure you're using `meteor npm install jstat --save` (note the addition of `npm` here, not in your original message).

The reason you're seeing the difference between client and server is jStat exports it's browser side differently than the server, as noted in it's documentation here: https://github.com/jstat/jstat#module-loaders

You might get a more consistent object across both using `import * as jStat from 'jstat';`, but I can't be sure.  Check with the package itself.

Lastly, in the future, please ask questions like this on either the [Meteor Forums](https://forums.meteor.com) or StackOverflow as Issues are reserved for documented bugs with reproductions.

**Comment by eric-burel** [comment]
Thanks, I did read that but had no idea how to work this out, and I never encountered the `import * as` syntax so far.

**Comment by corradio** [comment]
FYI what works for me is to directly import the module you need using `import { beta } from 'jStat';`

**Comment by eric-burel** [comment]
Hi, `import { normal } from 'jStat'` does set up jStat.normal on client correctly, but still not on server, jStat is undefined. This is giving me some headache but I'll figure this out ^^
